untitled/src/main/java/com/aca/homework/week18/website/service/core/PostService.java

```diff
@@ -0,0 +1,15 @@
+package com.aca.homework.week18.website.service.core;
+
+import com.aca.homework.week18.website.entity.Post;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface PostService {
+
+    Post create(CreatePostParams params);
+
+    Optional<Post> findById(Long id);
+
+    Optional<List<Post>> findAllByUserId(Long id);
```

Comment:
there is no need to wrap with optional, because the list es enough to describe the empty optional